src/location-picker.ts

```diff
@@ -39,8 +39,7 @@ export class LocationPicker implements EventTarget {
 
             this.countryCode = options.countryCode;
             this.defaultIcon = options.defaultIcon;
-            this.height = options.height;
-
+            this.height = (typeof options.height != "undefined") ? options.height :  this.height;
```

Comment:
True although because you use the typeof operator it always returns a string so it works as intended even if not syntax correct. Ill fix it ether way, this is actually changes I added on the branch in 2020 (I think?) which were never merged